File: tests/expected/fib.go
package main

import (
	"fmt"
)

func Fib(i int) int {
	if i == 0 || i == 1 {
		return 1
	}
	return (Fib((i - 1)) + Fib((i - 2)))
}

func main() {
	fmt.Printf("%v\n", Fib(5))
}

File: tests/expected/fib.zig
const std = @import("std");

fn print(comptime fmt: []const u8, args: anytype) void {
    const io = std.Io.Threaded.global_single_threaded.io();
    var buffer: [1024]u8 = undefined;
    var writer = std.Io.File.stdout().writerStreaming(io, &buffer);
    const out = &writer.interface;
    out.print(fmt, args) catch return;
    out.flush() catch return;
}
pub fn fib(i: i32) i32 {
    if (i == 0 or i == 1) {
        return 1;
    }

    return (fib((i - 1)) + fib((i - 2)));
}

pub fn main() void {
    print("{}\n", .{fib(5)});
}
